rules/windows/file/file_event/file_event_win_susp_lnk_double_extension.yml

```diff
@@ -35,6 +36,9 @@ detection:
         TargetFilename|contains:
             - '\AppData\Roaming\Microsoft\Office\Recent\'
             - '\AppData\Roaming\Microsoft\Windows\Recent\'
+            - '\AppData\Roaming\Microsoft\Excel'
+            - '\AppData\Roaming\Microsoft\PowerPoint'
+            - '\AppData\Roaming\Microsoft\Word'
```

Comment:
Better separate them and add image path that is creating them to be more specific.